enrollment-server/src/main/java/com/wultra/app/enrollmentserver/configuration/PowerAuthWebServiceConfiguration.java

```diff
@@ -38,8 +38,6 @@
 @Configuration
 @ConfigurationProperties("ext")
 @ComponentScan(basePackages = {
-        "com.wultra.app.docverify",
```

Comment:
One of the most crucial reasons, why onboarding components were initialized at onboarding. Moved to `onboardingserver` package to make it more obvious.